app/services/page_updater.rb

```diff
@@ -82,7 +82,11 @@ def update_page
     return unless @params[:page]
     plugins_before = @page.plugins
     @page.assign_attributes(@params[:page])
-    @page.save
+
+    if @page.save
+      QueueManager.push(@page, job_type: :update)
```

Comment:
you could make this only do this if the relevant fields changed, I think it's something like
```
if !(@page.changed & [:language_id, :slug, :id, :title]).empty?
  QueueManager.push(@page, job_type: :update)
end
```